http/handlers.go

```diff
@@ -78,13 +131,12 @@ func EncodeReply(_ context.Context, w http.ResponseWriter, rep interface{}) erro
 		return nil
 	}
 
-	w.Header().Set("Content-Type", "application/json; charset=utf-8")
-	w.WriteHeader(http.StatusOK)
-
-	var json, err = json.MarshalIndent(rep, "", " ")
-
-	if err == nil {
-		w.Write(json)
+	switch e := rep.(type) {
+	case GenericResponse:
```

Comment:
GenericResponse is HTTP oriented. 
The intial purpose was to split it between an answer that can be translated into a specific kind of response, HTTP or gRPC, ... I know that we don't have it anymore but I think it was a good split of concerns.